python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1200,3 +1200,41 @@ def test_groupby_nunique_series():
         gdf.groupby("a")["b"].nunique(),
         check_dtype=False,
     )
+
+
```

Comment:
Should we add a test or two that lists are properly excluded from other aggregations like `sum` for example when called generally?

Reply:
Added a test for `mean`